app/Voucher.php

```diff
@@ -174,28 +175,21 @@ public static function createRangeDefFromVoucherCodes($startCode, $endCode)
         return (object) $rangeDef;
     }
 
-    /**
-     * Determines if the given voucher range contains entries already delivered
-     *
-     * @param $rangeDef object { 'start', 'end', 'shortcode', 'sponsor_id' }
-     * @return bool
-     */
-    public static function rangeIsDeliverable($rangeDef)
+    public static function collectionHasVoidable($currentstates)
```

Comment:
this method doesn't seem to be used, neither are its argument `currentstates` and the declared variable (`$voidable_states`) 🤔

Reply:
I'll prune it